tests/core/config/startupFocus.js

```diff
@@ -0,0 +1,80 @@
+/* bender-tags: editor */
```

Comment:
Test file names should not be camel cased, so you would have `startupfocus` and `startupfocusend`.